velox/vector/DictionaryVector-inl.h

```diff
@@ -205,5 +186,34 @@ VectorPtr DictionaryVector<T>::slice(vector_size_t offset, vector_size_t length)
           *INTEGER(), indices_, offset, length, this->pool_));
 }
 
+template <typename T>
+void DictionaryVector<T>::validate(
+    const std::optional<BaseVector::ValidateOptions>& options) const {
+  BaseVector::validate(options);
```

Comment:
`SimpleVector::validate`

Always call the immediate base class method to avoid glitches when we add a override to the immediate base